src/components/FormElements/XdrPicker.tsx

```diff
@@ -5,11 +5,13 @@ interface XdrPickerProps extends Omit<TextareaProps, "fieldSize"> {
   id: string;
   fieldSize?: "sm" | "md" | "lg";
   labelSuffix?: string | React.ReactNode;
-  label: string;
+  label: string | React.ReactNode;
   value: string;
   placeholder?: string;
-  error: string | undefined;
-  onChange: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+  error?: string | undefined;
+  note?: string | React.ReactNode;
```

Comment:
using note as a success message

Reply:
:+1: We might need to add the `success` param in SDS.

Reply:
done!